test/Microsoft.IdentityModel.Tokens.Saml.Tests/ValidateTheoryData.cs

```diff
@@ -16,14 +15,14 @@ public static void AddValidateAudienceTheoryData(List<TokenTheoryData> theoryDat
                 Audiences = new List<string>(),
                 First = true,
                 ExpectedException = ExpectedException.ArgumentNullException("IDX10000:"),
-                TestId = "TokenValidationParameters null",
+                TestId = "TokenValidationParameters_Null",
```

Comment:
Instead of setting TestId this way - could you do it like so `new TokenTheoryData("TokenValidationParameters_Null")`? TokenTheoryData already has a ctor that takes a TestId and doing so this way allows `theoryData.CallContext.DebugId` to be set.